### PR TITLE
Add Forgotten Heroes pack

### PR DESCRIPTION
**Pull request by EvadLhorg**

### CODE REVIEW

File: restricted-list.json [hunk 1]
```diff
@@ -1521,7 +1521,27 @@
             "17157",
             "17158",
             "17159",
-            "17160"
+            "17160",
```
Comment (stopfstedt) [inline restricted-list.json:1524]: this suggests that the FH cards are banned only for the Conclave 2.0 RL. if i'm not mistaken then you would also want to back-fill the `bannedCards` lists for previous RLs, like the FFG era ones.

**Review by stopfstedt — CHANGES_REQUESTED** [review]
Needs some additional back-filling in the `restricted-list.json` file, otherwise this looks good to me.

File: restricted-list.json (hunk 1 above)
Comment (EvadLhorg) [reply restricted-list.json:1704]: You are absolutely right. We did the same for the bin pack cards and i´ve pushed the changes you requested

**Review by stopfstedt — APPROVED** [review]
LGTM

**Pull request merged**